test/TestData.hs

```diff
@@ -0,0 +1,64 @@
+{-# LANGUAGE DataKinds             #-}
+{-# LANGUAGE DeriveGeneric         #-}
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings     #-}
+{-# LANGUAGE RankNTypes            #-}
+{-# LANGUAGE TypeSynonymInstances  #-}
+
+
+module TestData where
+
+import           Data.Proxy
+import           Data.Text                                 (Text)
+import           Data.Typeable
+import           GHC.Generics                              (Generic)
+import           Language.PureScript.Bridge
+import           Language.PureScript.Bridge.PSTypes
+
+
+
+-- Check that examples compile:
+textBridge :: BridgePart
+textBridge = do
+   typeName ^== "Text"
+   typeModule ^== "Data.Text.Internal" <|> typeModule ^== "Data.Text.Internal.Lazy"
+   return psString
+
+stringBridge :: BridgePart
+stringBridge = do
+   haskType ^== mkTypeInfo (Proxy :: Proxy String)
+   return psString
```

Comment:
textBridge und stringBridge kommen eh wirklich auch im Code vor und werden in der defaultBridge verwendet.